docs/user-guide/side-bar.md

```diff
@@ -39,6 +39,9 @@ The *3D navigation* in [MapStore](https://mapstore.geosolutionsgroup.com/mapstor
 
 <img src="../img/side-bar/3D-mode.jpg" class="ms-docimage"/>
 
+!!! note
+    The *3D navigation* support also the **3D Tiles** once is added to the *Catalog tool* as explained [here](catalog.md#3d-tiles-catalog).
```

Comment:
The *3D mode* in MapStoere support also the rendering **3D Tiles** layers once they are added through the *Catalog tool* as explained [here](catalog.md#3d-tiles-catalog).